### PR TITLE
Fixes for tests

### PR DESCRIPTION
**Pull request by GustavoLCR**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Running tests was leaving some bins because trying to remove them while they were open doesn't work on windows

* Close files before trying to delete them
* Add `-N` to not load the rizinrc in `read from stdin` test to avoid overwriting the color setting

**Test plan**

Tests pass

**Closing issues**

None

### CODE REVIEW

File: test/db/cmd/write [hunk 1]
```diff
@@ -300,7 +300,6 @@ NAME=wci should commit the changes to the file
 FILE=malloc://1024
 BROKEN=1
 CMDS=<<EOF
-!rm -rf .tmp
```
Comment (ret2libc) [inline test/db/cmd/write:303]: why this? what if for some reasons `.tmp` already exists?

File: test/db/cmd/write [hunk 2]
```diff
@@ -309,6 +308,9 @@ wx ff
 wci
 e.src.null=true
 !!rz-diff .tmp/.b.exe.orig .tmp/.b.exe.new
+o--
+rm .tmp/.b.exe.orig
+rm .tmp/.b.exe.new
```
Comment (ret2libc) [inline test/db/cmd/write:313]: should you remove `.tmp` too?

Comment (GustavoLCR) [reply test/db/cmd/write:313]: This folder is used by many tests, sometimes simultaneously so removing it inside a test will rarely work

File: test/db/cmd/write (hunk 1 above)
Comment (GustavoLCR) [reply test/db/cmd/write:303]: same reason as https://github.com/rizinorg/rizin/pull/2818#discussion_r924193289

**Review by ret2libc — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Pull request merged**